### ISSUE TITLE
Changes in Poedit break NVDA specific features

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#7303]
### Steps to reproduce:
1. [Install Poedit2](https://poedit.net/news/introducing-poedit2)
2. Open a po file
3. Select a fuzzy or new message

### Expected behavior:
NVDA plays a beep, indicating that an item is fuzzy or missing (i.e. the item is bold)

### Actual behavior:
No indication of fuzzy/new messages

I did not check the translator comments window, but I assume that window is broken as well, since a major restyle of the dialog has taken place. It seems they are using WX now.

**Comment by vslavik** [comment]
"They" (that would be me) were using wxWidgets since always, the translator comments window is unchanged since 1.8, but the two relevant changes are:

1. the list (which is no longer ListView, but a custom control that emulates ListView for accessibility and its accessibility "appearance" may be slightly different)
2. the "fuzzy" toolbar button was replaced with a "needs work" toggle button in the editing area

I do have better flexibility in the list and can tweak list items (accessibility) presentation if that would make things simpler — just let me know if you need my cooperation.

**Comment by ehollig** [comment]
@leonardder is this still an issue?

**Comment by bdorer** [comment]
yes, this issue still exists. Furthermore if I tab from translation edit 
field in to the string list, NVDA should read the currently selected 
list entry, not the one at the top of the list. Furthermore I'd like to 
have a way to read poedits translation proposals.

Am 19.03.2019 um 04:07 schrieb Ethan Holliger:

**Comment by bdorer** [comment]
So far I've seen there is allready Poedit issue 415 regarding tab navigation for static text (was read-only field before). I created Issues 693 to reclass needs work button as a checkbox and 694 to get more accessibility for translations list. Just fyi.

**Comment by pvagner** [comment]
I have been pointed to this by one of my friends also doing translation work.
It turns out NVDA's poedit app module is entirely useless with the recent poedit versions.
I haven't yet looked at all the poedit related accessibility issues but since they have done loads of accessibility related work it would be super cool if they were able to make those controls focusable so we can access both comments as well as translator notes.

**Comment by leonardder** [comment]
In poedit 3, it looks like they switched back to something that feels more like the poedit 1 UI.
As I got hopelessly frustrated by this issue and it seems to hold back #14285 (i.e. enabling UIA for RichEdit when advertised), I'll see what I can do to bring the poedit app module in line with most recent version.

**Comment by leonardder** [comment]
I had a look into this. We'd be perfectly able to redesign the appmodule against the new version, were it not that the control identifiers change on every release, so we couldn't hardcode these. That would mean we'd need to do object walking, but the object order seems to change based on whether there are translation notes and/or comments.
